portfolio/src/main/webapp/script.js

```diff
@@ -49,10 +54,26 @@ function showSlides(newIndex) {
   dots[slideIndex].className += ' active';
 }
 
+/** Called when comment number limit selector changes. */
+function changeLimit() {
+  const limit = document.getElementById('select-comment-number').value;
+  commentLimit = limit;
+  showComments();
+}
+
+/** Called when sorting order selector changes. */
```

Comment:
You should explain what the function does.- so in this case changes sorting order.